### PR TITLE
Enable running shared-components locally

### PR DESCRIPTION
**Pull request by clottman**
- adds dotenv so people working on this locally can set the port value in .env
- replaces hard-coded `app` path with `path.resolve(__dirname, ...` so that the app can run on local machines as well as on Glitch

- To test this locally while Glitch is down, comment out the `glitchButton` code in public/index.html. 
- Will create a remix with this branch on it when the site's back up.

### CODE REVIEW

**Comment by clottman** [comment]
Updated readme, and got this running on glitch at ~boundless-olive to make sure it didn't break anything.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ This package also renders its own documentation and development environment.
 
 **TODO** explain how glitch remix / pull request flow works, use docs from community site
 
+### Running off Glitch
```
Comment (ehmorris) [inline README.md]: Nice! I might call this "Running locally" but I don't have a strong preference.

**Comment by ttseng** [comment]
awesome!  it's working for me.

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**